source/geppetto/4.2/index.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: default
+title: "Geppetto"
+subtitle: "Introduction to Geppetto"
+canonical: "/geppetto/latest/geppetto.html"
+---
+[geppetto_install]: ./images/geppetto_install.png
+[IDE_overview]: ./images/IDE_overview.png
+[prev]: ./images/geppettoprev.png
+[add]: ./images/geppettoaddpuppetdb.png
+[events]: ./images/geppettoevents.png
```

Comment:
This `events` reference is never used in the text, and the file doesn't seem to exist; it seems the one referenced by `prev` shows the events view and is displayed at the proper place. The `events` reference can probably be removed, unless I'm missing something.